### DISCUSSION TITLE
在Linux系统中mqtt内存异常，使用net5.0版本

### DISCUSSION THREAD

**Discussion by huailiangwu** [dotnet/MQTTnet#1817] [General]
![image](https://github.com/dotnet/MQTTnet/assets/24580476/9925f4aa-d346-494e-8dde-004fd361d268)
![1692266640610](https://github.com/dotnet/MQTTnet/assets/24580476/078a33ca-b382-441c-8f63-bcbb58b12e27)

mqtt版本：MQTTnet.AspNetCore 4.2.0.706
 服务部署在k8s中，使用长连接，程序运行后，内存一直增加，会是什么原因啊？

**Comment by gfoidl** [comment]
@huailiangwu can you please post this in english (so that everyone can read it)?

**Reply by huailiangwu** [reply]
Mqtt memory leak in Linux system, using net5.0 version # 1817

Mqtt version: MQTTnet.AspNetCore 4.2.0.706

The service is deployed in K8S and uses long connections. After the program runs, the memory keeps increasing. What is the reason?

**Comment by huailiangwu** [comment]
@gfoidl Please take a look at the reason?

**Reply by gfoidl** [reply]
In https://github.com/dotnet/MQTTnet/discussions/1817#discussioncomment-6751903 I was just "moderating".

Did you take a memory profile? So you could see what objects are in the growing heap.

**Reply by huailiangwu** [reply]
Can you provide a demo of quartz. net without adding it?

**Reply by gfoidl** [reply]
What has Quartz.net to do with this discussion?
So, no I can't provide such a demo.
